js/src/lib/components/InferenceWidget/shared/helpers.ts

```diff
@@ -87,9 +88,9 @@ async function callApi(
 	const body: File | string = 'file' in requestBody
 		? requestBody.file
 		: JSON.stringify(requestBody);
-	
+
 	return await fetch(
-		`${url}/models/${repoId}`,
+		`${url}${appendRepoPath ? `/models/${repoId}` : ''}`,
```

Comment:
Not sure I understand here; is it because `url` will already include the model path in the case of IE?

Reply:
We don't have a "model path" in IE. Each endpoint only has a single API point (the `url` arg).